test/Serilog.ApprovalTests/ApiApprovalTests.cs

```diff
@@ -16,6 +16,11 @@ public void PublicApi_Should_Not_Change_Unintentionally()
                 ExcludeAttributes = new[] { "System.Diagnostics.DebuggerDisplayAttribute" },
             });
 
-        publicApi.ShouldMatchApproved(options => options.WithFilenameGenerator((_, _, fileType, fileExtension) => $"{assembly.GetName().Name!}.{fileType}.{fileExtension}"));
+        publicApi.ShouldMatchApproved(options =>
+        {
+            // Comment this line out to view the failure as a diff. Leave it here so that CI builds don't hang when this test fails.
+            options.NoDiff();
```

Comment:
How does hang look like? I do not use `NoDiff` in other repos and did not see issues.

Reply:
That's a good question, I may have jumped the gun here, based on experience with other tools that launch windowed executables for this.